packages/labs/src/components/tooltip/tooltip2.tsx

```diff
@@ -0,0 +1,169 @@
+     * controlled mode, where the only way to change visibility is by updating this property.
+     * @default undefined
+     */
+    isOpen?: boolean;
+
+    /**
+     * Whether to apply minimal styles to this tooltip, which includes removing the arrow
+     * and adding the `.pt-minimal` class to minimize and accelerate the transitions.
+     * @default false
+     */
+    minimal?: boolean;
+
+    /**
+     * Popper modifier options, passed directly to internal Popper instance.
+     * See https://popper.js.org/popper-documentation.html#modifiers for complete details.
+     */
+    modifiers?: PopperModifiers;
+
+    /**
+     * Callback invoked in controlled mode when the tooltip open state *would* change due to
+     * user interaction.
+     */
+    onInteraction?: (nextOpenState: boolean) => void;
+
+    /**
+     * Whether the tooltip should open when its target is focused.
+     * If `true`, target will render with `tabindex="0"` to make it focusable via keyboard navigation.
+     * @default: true
+     */
+    openOnTargetFocus?: boolean;
+
+    /**
+     * Space-delimited string of class names applied to the
+     * portal which holds the tooltip if `inline` is set to `false`.
+     */
+    portalClassName?: string;
+
+    /**
+     * The position (relative to the target) at which the popover should appear.
+     * The default value of `"auto"` will choose the best placement when opened and will allow
+     * the popover to reposition itself to remain onscreen as the user scrolls around.
+     * @default "auto"
+     */
+    placement?: Placement;
+
+    /**
+     * The name of the HTML tag to use when rendering the tooltip target wrapper element.
+     * @default "span"
+     */
+    rootElementTag?: string;
+
+    /**
+     * A space-delimited string of class names that are applied to the tooltip.
+     */
+    tooltipClassName?: string;
+
+    /**
+     * Indicates how long (in milliseconds) the tooltip's appear/disappear transition takes.
+     * This is used by React `CSSTransitionGroup` to know when a transition completes
+     * and must match the duration of the animation in CSS.
+     * Only set this prop if you override Blueprint's default transitions with new transitions of a different length.
+     * @default 100
+     */
+    transitionDuration?: number;
+}
+
+@PureRender
+export class Tooltip2 extends React.Component<ITooltip2Props, {}> {
+    public static displayName = "Blueprint.Tooltip2";
+
+    public static defaultProps: Partial<ITooltip2Props> = {
+        defaultIsOpen: false,
+        disabled: false,
+        hoverCloseDelay: 0,
+        hoverOpenDelay: 100,
+        inheritDarkTheme: true,
+        inline: false,
+        isOpen: false,
+        minimal: false,
+        openOnTargetFocus: true,
+        placement: "auto",
+        rootElementTag: "span",
+        transitionDuration: 100,
+    };
+
+    public render() {
+        const { children, intent, tooltipClassName } = this.props;
```

Comment:
you could do the following to avoid passing unsupported props?
```tsx
const { children, intent, tooltipClassName, ...restProps } = this.props;
<Tooltip2 {...restProps} ... />
```